app/www/styles/profile.less

```diff
@@ -3,14 +3,193 @@
 }
 
 .profile-avatar-img {
-    width: 60%;
+    width: 80%;
     margin: 0 auto;
 }
 
 .profile-details {
     width: 50%;
+    font-family: titleFont;
 }
 
 .profile-detail {
-    font-size: 150%;
-}
\ No newline at end of file
+    font-size: 40px;
+    font-family: menuFont;
+    color: #709CD5;
+}
+.profile-progress{
```

Comment:
Fix indentation for the rest of the file: everything within a rule should be 4-space indented. I think Atom has a feature for auto-formatting if I'm not mistaken!